src/api/api.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+
+// export const getGreetings = async () => {
+//    const response = await fetch("api/v1/greetings");
+
+//    return response.json();
+// };
```

Comment:
- [ ] Please remove every commented code from your files as it makes your code unreadable and messy. :pray:

Reply:
:heavy_check_mark: Done !